frontends/mit-open/src/page-components/LearningResourceDrawer/LearningResourceDrawer.tsx

```diff
@@ -1,17 +1,43 @@
-import React from "react"
+import React, { useEffect } from "react"
 import { RoutedDrawer, ExpandedLearningResourceDisplay } from "ol-components"
 import type { RoutedDrawerProps } from "ol-components"
 import { useLearningResourcesDetail } from "api/hooks/learningResources"
 import { imgConfigs } from "@/common/constants"
 import { useSearchParams } from "react-router-dom"
 import { RESOURCE_DRAWER_QUERY_PARAM } from "@/common/urls"
+import { usePostHog } from "posthog-js/react"
 
 const RESOURCE_DRAWER_PARAMS = [RESOURCE_DRAWER_QUERY_PARAM] as const
 
+const useCapturePageView = (resourceId: number) => {
+  const { data, isSuccess } = useLearningResourcesDetail(Number(resourceId))
+  const posthog = usePostHog()
+
+  useEffect(() => {
+    if (!APP_SETTINGS.posthog?.enabled) return
+    if (!isSuccess) return
+    posthog.capture("lrd_view", {
+      resourceId: data?.id || "unknown",
```

Comment:
I meant to say this earlier, but I would be extraordinarily surprised if `"unknown"` ever came through.